vnext/Desktop.IntegrationTests/DesktopTestRunner.cpp

```diff
@@ -44,35 +45,30 @@ shared_ptr<ITestInstance> TestRunner::GetInstance(
   devSettings->jsiRuntimeHolder = std::make_shared<ChakraRuntimeHolder>(devSettings, jsQueue, nullptr, nullptr);
 
   vector<tuple<string, CxxModule::Provider, shared_ptr<MessageQueueThread>>> extraModules{
-      std::make_tuple(
-          "AsyncLocalStorage",
-          []() -> unique_ptr<CxxModule> {
-            return /*CreateAsyncStorageModule(L"ReactNativeAsyncStorage")*/ nullptr;
-          }, // #6882
-          nativeQueue),
-      std::make_tuple(
-          "WebSocketModule",
-          []() -> unique_ptr<CxxModule> { return std::make_unique<WebSocketModule>(); },
-          nativeQueue),
-      std::make_tuple(
-          "Networking",
-          []() -> unique_ptr<CxxModule> { return std::make_unique<Microsoft::React::NetworkingModule>(); },
-          nativeQueue),
-      std::make_tuple(
-          "Timing", [nativeQueue]() -> unique_ptr<CxxModule> { return CreateTimingModule(nativeQueue); }, nativeQueue),
+      {"AsyncLocalStorage",
+       []() -> unique_ptr<CxxModule> {
+         return /*CreateAsyncStorageModule(L"ReactNativeAsyncStorage")*/ nullptr; // #6882
+       },
+       nativeQueue},
+      {"WebSocketModule", []() -> unique_ptr<CxxModule> { return std::make_unique<WebSocketModule>(); }, nativeQueue},
+      {"Networking",
+       []() -> unique_ptr<CxxModule> { return std::make_unique<Microsoft::React::NetworkingModule>(); },
+       nativeQueue},
+      {"Timing", [nativeQueue]() -> unique_ptr<CxxModule> { return CreateTimingModule(nativeQueue); }, nativeQueue},
       // Apparently mandatory for /IntegrationTests
-      std::make_tuple(
-          TestAppStateModule::name,
-          []() -> unique_ptr<CxxModule> { return std::make_unique<TestAppStateModule>(); },
-          nativeQueue),
+      {TestAppStateModule::name,
+       []() -> unique_ptr<CxxModule> { return std::make_unique<TestAppStateModule>(); },
+       nativeQueue},
       // Apparently mandatory for /IntegrationTests
-      std::make_tuple(
-          "UIManager", []() -> unique_ptr<CxxModule> { return std::make_unique<TestUIManager>(); }, nativeQueue),
+      {"UIManager", []() -> unique_ptr<CxxModule> { return std::make_unique<TestUIManager>(); }, nativeQueue},
       // Apparently mandatory for /IntegrationTests
-      std::make_tuple(
-          TestDeviceInfoModule::name,
-          []() -> unique_ptr<CxxModule> { return std::make_unique<TestDeviceInfoModule>(); },
-          nativeQueue)};
+      {TestDeviceInfoModule::name,
+       []() -> unique_ptr<CxxModule> { return std::make_unique<TestDeviceInfoModule>(); },
+       nativeQueue},
+
```

Comment:
nit: should we remove the extra empty line to match the rest of the code?

Reply:
I think I'd actually add the line in the previous entries.
It improves readability.